### ISSUE TITLE
Looks like that adding normalize.css to several apps causes build problems

### ISSUE THREAD

**Issue by ferrannp** [TrueCar/gluestick#903]
We experienced that internally. Regardless of if we should add `normalize.css` to Gluestick or not (in my opinion no), we need to find out why is this causing problems.

**Comment by toddw** [comment]
Agreed! We don't need to include normalize.css but we do need to resolve the issue

**Comment by zamotany** [comment]
I don't think it's the issue with `compiled` alias - I've builded and run server with alias (otherwise it would crash) in production, removed `compiled` from `normalize.css` imports and builded client and the issue is still present.

**Comment by zamotany** [comment]
I believe the best way to go is to try to reproduce it with bare `webpack.config.js` and w/o gluestick.

Update: can't reproduce the issue with 2 apps with gluestick

**Comment by supergibbs** [comment]
Maybe a different issue but I removed `normalize.css` from my app because I am using Bootstrap4 and it has its own css reset but when I updated gluestick it re-added `normalize.css` because it thinks it's required even though I am not using it.

**Comment by zamotany** [comment]
The default project template usues `normalize.css`, so it's in `package.json` template. When you run `auto-upgrade` it will compare your `package.json` and template's one then prompt to install missing dependencies or update them. That's why you will have `normalize.css` added as a dependency.
